e2e_playwright/multipage_apps/mpa_basics_test.py

```diff
@@ -76,13 +76,6 @@ def test_can_switch_to_the_first_page_with_a_duplicate_name(app: Page):
     expect(app.get_by_test_id("stHeading")).to_contain_text("Page 4")
 
 
-def test_can_switch_to_the_second_page_with_a_duplicate_name(app: Page):
-    """Test that we can switch to the second page with a duplicate name."""
-    app.get_by_test_id("stSidebarNav").locator("a").nth(4).click()
-    wait_for_app_run(app)
-    expect(app.get_by_test_id("stHeading")).to_contain_text("Page 5")
-
-
```

Comment:
We effectively need to disallow duplicate page names

Reply:
Another approach would be to keep track of page names when reading the file names from the directory and prefix them internally or so with their index. Would that work?